relayer/__init__.py

```diff
@@ -28,7 +29,7 @@ class Relayer(object):
             for http://kafka-python.readthedocs.io/en/master/apidoc/KafkaProducer.html
     """
 
-    def __init__(self, logging_topic, **kwargs):
+    def __init__(self, logging_topic: str, **kwargs: str) -> None:
```

Comment:
Did you tried as it's referred here http://mypy.readthedocs.io/en/latest/kinds_of_types.html#callable-types-and-lambdas ?
```
(*args: Any, **kwargs: Any)
```

Reply:
Yes, I used `(*args: Any, **kwargs: Any)` for other method. In this case since we know that all arguments will be `str`, it is better to use this typing, since it is more specific.